index.bs

```diff
@@ -1499,6 +1640,54 @@ partial interface MLGraphBuilder {
     tensor is specified by the *newShape* argument.
 </div>
 
+### setOperand ### {#api-mlgraphbuilder-set-operand}
```

Comment:
I suppose `setOperand` should be the implementation detail. 

It is because when building a graph, if the user code wants to fuse the activation, it would call the `MLOperator` version, e.g. `output = builder.conv2d(input, filter, {activation: builder.relu()})`. Otherwise, the caller would call the `MLOperand` version, e.g. `output = builder.relu(builder.conv2d(input, filter))`. The `setOperand` seems not useful for user code.

Any other usages I missed?

Reply:
Yes, in operations like `batchNorm` or `gruCell`. You're right it could be treated as a hidden concept for atomic operations like `conv2d`. For composite operations that happen to accept fused operators, it should be possible to implement them purely as composites of smaller ops, as the pseudo-code in their note sections suggests.

Reply:
How about a name like ```apply``` instead of ```setOperand```? I think it is better if we use a name that indicates the side-effect-free functional behavior rather than one that suggests an imperative behavior with side-effects. I think this is essentially the operation of applying a function (represented by operator) to its arguments.

Reply:
Thanks for the feedback @gramalingam. I like that.